src/sentry/integrations/slack/threads/activity_notifications.py

```diff
@@ -178,6 +178,13 @@ class ExternalIssueCreatedActivityNotification(GroupActivityNotification):
     metrics_key = "create_issue"
     title = "External Issue Created"
 
+    def format_provider_name(provider: str) -> str:
+        # Make sure to make the proper noun have correct capitalization
+        # Take in account special cases
+        # I.e. github -> GitHub, jira -> Jira
+        special_cases = {"github": "GitHub"}
+        return special_cases.get(provider.lower(), provider.capitalize())
```

Comment:
instead of doign this, can we just update `_external_issue_activity_factory` and update it to return a special Github class?